manifests/php.yml

```diff
@@ -5,23 +5,129 @@ tests/:
+        Test_Schema_Request_Json_Body: missing_feature
+        Test_Schema_Request_Path_Parameters: missing_feature
+        Test_Schema_Request_Query_Parameters: v0.94.0
+        Test_Schema_Response_Body: missing_feature
+        Test_Schema_Response_Headers: v0.94.0
     iast/:
       sink/:
+        test_command_injection.py:
+          TestCommandInjection: missing_feature
+        test_hardcoded_secrets.py:
+          Test_HardcodedSecrets: missing_feature          
+        test_header_injection.py:
+          TestHeaderInjection: missing_feature
+        test_hsts_missing_header.py: 
+          Test_HstsMissingHeader: missing_feature
+        test_insecure_cookie.py:
+          TestInsecureCookie: missing_feature
+        test_ldap_injection.py:
+          TestLDAPInjection: missing_feature
+        test_no_httponly_cookie.py:
+          TestNoHttponlyCookie: missing_feature
+        test_no_samesite_cookie.py:
+          TestNoSamesiteCookie: missing_feature
+        test_nosql_mongodb_injection.py:
+          TestNoSqlMongodbInjection: missing_feature
+        test_path_traversal.py:
+          TestPathTraversal: missing_feature
+        test_sql_injection.py:
+          TestSqlInjection: missing_feature
+        test_ssrf.py:
+          TestSSRF: missing_feature
+        test_trust_boundary_violation.py:
+          Test_TrustBoundaryViolation: missing_feature
+        test_unvalidated_redirect.py:
+          TestUnvalidatedHeader: missing_feature
+          TestUnvalidatedRedirect: missing_feature
+        test_unvalidated_redirect_forward.py:
+          TestUnvalidatedForward: missing_feature
+        test_weak_cipher.py:
+          TestWeakCipher: missing_feature
+        test_weak_hash.py:
+          TestWeakHash: missing_feature
+        test_weak_randomness.py:
+          TestWeakRandomness: missing_feature
+        test_xcontent_sniffing.py:
+          Test_XContentSniffing: missing_feature        
+        test_xpath_injection.py:
+          TestXPathInjection: missing_feature
         test_xss.py:
           TestXSS:
             '*': missing_feature
+      source/:
+        test_body.py:
+          TestRequestBody: missing_feature
+        test_cookie_name.py:
+          TestCookieName: missing_feature
+        test_cookie_value.py:
+          TestCookieValue: missing_feature
+        test_header_name.py:
+          TestHeaderName: missing_feature
+        test_header_value.py:
+          TestHeaderValue: missing_feature
+        test_multipart.py:
+          TestMultipart: missing_feature
+        test_parameter_name.py:
+          TestParameterName: missing_feature
+        test_parameter_value.py:
+          TestParameterValue: missing_feature
+        test_path.py:
+          TestPath: missing_feature
+        test_uri.py:
+          TestURI: missing_feature  
     waf/:
       test_addresses.py:
         Test_BodyJson: missing_feature
+        #Test_BodyRaw: v0.1.0 #I don't know first release
```

Comment:
Can you ask @Anilm3 a map `appsec_version -> tracer_version` ?

Reply:
Hopefully this helps.

```
Appsec Version | Tracer Version
--             | --
v0.3.0         | 0.73.0
v0.3.2         | 0.74.0
v0.4.0         | 0.78.0
v0.4.3         | 0.80.0
v0.4.4         | 0.81.0
v0.6.0         | 0.84.0
v0.7.0         | 0.86.0
v0.7.1         | 0.86.2
v0.7.2         | 0.86.3
v0.8.0         | 0.87.0
v0.8.1         | 0.87.2
v0.9.0         | 0.88.0
v0.10.0        | 0.89.0
v0.13.1        | 0.91.0
v0.14.0        | 0.92.0
v0.14.1        | 0.92.1
v0.14.2        | 0.92.2
v0.14.3        | 0.93.0
v0.95.0        | 0.95.0

Reply:
Ok. Done!